mangaki/mangaki/templates/mangaki/reco_list.html

```diff
@@ -11,29 +11,13 @@
             Notez-les puis rechargez la page pour en faire apparaître
             d'autres.
 
-    Inclure la liste des "À voir" dans les recommandations : <input type="checkbox" name="reco_willsee_ok" {% if user.profile.reco_willsee_ok %} checked{% endif %}/>
-    <form name="editor_choice">
-      <div>
-      Filtrer par éditeur :
```

Comment:
This is finally leaving (even though I'm not sure I see the link with this PR...) \o/